src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Wysiwyg.Edit.cshtml

```diff
@@ -1,16 +1,20 @@
 @model OrchardCore.ContentFields.ViewModels.EditHtmlFieldViewModel
+@using OrchardCore.ContentManagement.Metadata.Models
+@using OrchardCore.ContentFields.Settings;
 
 @{
-    var settings = Model.PartFieldDefinition.GetSettings<HtmlFieldSettings>();
-    var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
+    var settings = Model.PartFieldDefinition.Settings.ToObject<HtmlFieldSettings>();
+    var wysiwygSettings = Model.PartFieldDefinition.GetSettings<HtmlFieldWysiwygEditorSettings>();
+    //var culture = await Orchard.GetContentCultureAsync(Model.ContentItem);
 }
 
-<script name="trumbowyg" depends-on="admin" asp-src="~/OrchardCore.Resources/Scripts/trumbowyg.min.js" debug-src="~/OrchardCore.Resources/Scripts/trumbowyg.js" at="Foot"></script>
-<style name="trumbowyg" asp-src="~/OrchardCore.Resources/Styles/trumbowyg.min.css" debug-src="~/OrchardCore.Resources/Styles/trumbowyg.css"></style>
+<script asp-name="trumbowyg" depends-on="admin" asp-src="~/OrchardCore.Resources/Scripts/trumbowyg.min.js" debug-src="/OrchardCore.Resources/Scripts/trumbowyg.js" at="Foot"></script>
+<style asp-src="~/OrchardCore.Resources/Styles/trumbowyg.min.css" debug-src="~/OrchardCore.Resources/Styles/trumbowyg.css"></style>
 
-<fieldset class="form-group">
+<div class="form-group">
     <label asp-for="Html">@Model.PartFieldDefinition.DisplayName()</label>
-    @if (culture.IsRightToLeft())
+    <textarea asp-for="Html" class="form-control"></textarea>
+    @*@if (culture.IsRightToLeft())
     {
```

Comment:
and this maybe needs to be added back in? (uncommented)

Reply:
Copied from the HtmlPart but I don't know how to retrieve the culture in the Field, because it doesn't have a Model.ContentItem like the Part.

Reply:
cool, this might/should work `var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);`

Reply:
I will try to retrieve it this way but I am not sure it will work:
`var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);`